lib/github_fetcher.rb

```diff
@@ -151,15 +151,34 @@ def ignored_ci_repos
     YAML.load_file(File.join(File.dirname(__FILE__), "../ignored_ci_repos.yml"))
   end
 
+  def is_ruby_app?(repo)
+    github.contents(repo, path: "Gemfile")
+  end
+
   def has_sca_sast_scans?(repo)
     return true if ignored_ci_repos.include?(repo)
 
-    ci_file = Base64.decode64(github.contents("#{organisation}/#{repo}", path: ".github/workflows/ci.yml").content)
+    ci_file = fetch_ci_file_content(repo)
+    return true if ci_file.nil? # if a CI file is not present assume no scans are needed
+
     sca_string = "uses: alphagov/govuk-infrastructure/.github/workflows/dependency-review.yml@main"
     sast_string = "uses: alphagov/govuk-infrastructure/.github/workflows/codeql-analysis.yml@main"
 
-    ci_file.include?(sca_string) && ci_file.include?(sast_string)
+    if is_ruby_app?(repo)
+      brakeman_string = "uses: alphagov/govuk-infrastructure/.github/workflows/brakeman.yml@main"
```

Comment:
Can we also check for `bundle exec brakeman` string rather than excluding a repo from the CI checks?